Code/Editor/Plugins/ProjectSettingsTool/ProjectSettingsToolWindow.cpp

```diff
@@ -647,33 +648,38 @@ namespace ProjectSettingsTool
         // iOS can be disabled if the plist file is missing
         if (platformId == PlatformId::Ios)
         {
-            const AZStd::string filename = m_projectRoot + PlatformResourcesFolder(platformId);
-            return CFileUtil::FileExists(filename.c_str());
+            AZStd::string plistPath = GetPlatformResource(platformId);
+            return !plistPath.empty();
         }
 
         return true;
     }
 
-    const char* ProjectSettingsToolWindow::PlatformResourcesFolder(PlatformId platformId)
+    AZStd::string ProjectSettingsToolWindow::GetPlatformResource(PlatformId platformId)
     {
         if (platformId == PlatformId::Ios)
         {
-            const AZStd::string firstfilename = m_projectRoot + "/Gem/Resources/Platform/iOS/Info.plist";
-            if (CFileUtil::FileExists(firstfilename.c_str()))
-            {
-                return "/Gem/Resources/Platform/iOS/Info.plist";
-            }
-            else
+            const char* searchPaths[] = {
+                "Resources/Platform/iOS/Info.plist",
+
+                // legacy paths
+                "Gem/Resources/Platform/iOS/Info.plist",
+                "Gem/Resources/IOSLauncher/Info.plist",
+            };
+
+            for (auto relPath : searchPaths)
             {
-                const AZStd::string filename = m_projectRoot + "/Gem/Resources/IOSLauncher/Info.plist";
-                if (CFileUtil::FileExists(filename.c_str()))
+                AZ::IO::FixedMaxPath projectPlist{ m_projectRoot };
+                projectPlist /= relPath;
+
+                if (CFileUtil::FileExists(projectPlist.c_str()))
                 {
-                    return "/Gem/Resources/IOSLauncher/Info.plist";
+                    return projectPlist.LexicallyNormal().c_str();
```

Comment:
A slightly more efficient operations is to call `.String()`
```suggestion
                    return projectPlist.LexicallyNormal().String();
```